_output/template_modifications/public/bdapi_bb_code_tag_attach.json

```
@@ -1,7 +1,7 @@
 {
     "template": "bb_code_tag_attach",
     "description": "Use api link for attachment data.",
-    "execution_order": 10,
+    "execution_order": 9000,
```

```suggestion
    "execution_order": 9999,
```

Chơi luôn 9999 cho máu chế.